en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -64,354 +60,352 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-5.png" alt="A screenshot of a card with dimensions 472 x 768. The card has a thick black border. The top of the card features a black and white image of a brick church with a large wooden door and branches of a tree that is out of frame. Under the image is, in bold print, 1851. Under this date is the text, 'The cornerstone of Sandwich First Baptist Church is laid on land donated by the Crown'" caption="Figure 5. The bolded text corresponds to the variable over which the mouse cursor is currently hovering." %}
 
-_nanDECK_ has a very useful ability to calculate the items in a spreadsheet of a designated value. It does this using an expression that is designated by curly brakets. In this case, _nanDECK_ translates the expression {(IMAGES)} to a caculation of the number of items with a field of an IMAGE. In our example, there are 58 images and so {(IMAGES)} equals 58. Knowing this, we can now understand that this line of script instructs nanDECK to print the images from rows 1 through 58.
+nanDECK can very usefully calculate the number of items listed under a designated field in a spreadsheet. It does this using an expression delimited by curly brakets. In this case, the expression {(IMAGES)} instructs nanDECK to caculate the number of items listed under the `IMAGE` field. Here, there are 58 images, so {(IMAGES)} returns `58`. Knowing this, you'll see that at line 6 of Figure 5 above, `IMAGE="1-{(IMAGES)}"` instructs nanDECK to print the images from rows 1 through 58.
 
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-6.png" alt="A screenshot of a part of an Excel spreadsheet with three columns. In column A, the first four lines have years but the next three entries have none. The middle column features the description of an event to be printed on the card. The last column lists the file location of the image for the card." caption="Figure 6: A card generated using nanDeck." %}
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-6.png" alt="A screenshot of a part of an Excel spreadsheet with three columns. In column A, the first four lines have years but the next three entries have none. The middle column features the description of an event to be printed on the card. The last column lists the file location of the image for the card." caption="Figure 6. A card generated using nanDeck." %}
```

Comment:
Here as well, the alt-text does not seem to match the image. Actually, I'm also unsure whether this is the right image here?

Reply:
The alt-text for Figure 6. should be, "A screenshot of a card with dimensions 472 x 768. The card has a thick black border. The top of the card features a black and white image of a brick church with a large wooden door and branches of a tree that is out of frame. Under the image is, in bold print, 1851. Under this date is the text, "The cornerstone of Sandwich First Baptist Church is laid on land donated by the Crown".